### DISCUSSION TITLE
Why does toss or tossStack crash the bot?

### DISCUSSION THREAD

**Discussion by vadimpashkov** [PrismarineJS/mineflayer#3141] [Q&A]
**Code:**
![image](https://github.com/PrismarineJS/mineflayer/assets/41518990/8ec4535f-8c26-4305-ab1c-d7899ee0d233)

**Error:**
<img width="898" alt="image" src="https://github.com/PrismarineJS/mineflayer/assets/41518990/a9036841-ce30-470c-939d-795c7a8b00b7">

**_Minecraft version 1.20.1_**

**Comment by PondWader** [comment]
Update Mineflayer, it was due to a change in the prismarine-windows API which was released as a minor version bump for some reason.
